src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1343,6 +1344,7 @@ internal enum ErrorCode
         #region diagnostics for local functions introduced in C# 7
         ERR_DynamicLocalFunctionParamsParameter = 8108,
         ERR_ExpressionTreeContainsLocalFunction = 8110,
+        ERR_LocalFunctionMissingBody = 8112,
```

Comment:
>ERR_LocalFunctionMissingBody = 8112, [](start = 8, length = 36)

This value violates the sort order, please move this member so that the sort order is restored.